packages/spotlight/bin/run.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 const { setupSidecar } = await import('../dist/sidecar.js');
-const port = process.argv[2];
+const port = process.argv.length >= 3 ? Number(process.argv[2]) : undefined;
```

Comment:
good catch!

Reply:
yes, but it was done by @dcramer initially in another PR. I thought its better to add it here as well.

Reply:
would be great if we could figure out how to dedupe this script, but this is beyond my domain knowledge w/ npm packages